packages/shared/src/hooks/useEnableNotification.ts

```diff
@@ -72,7 +72,7 @@ export const useEnableNotification = ({
     !isInitialized,
     !isNotificationSupported,
     (isSubscribed || hasPermissionCache) && !isEnabled,
-    hasEnabled && source === NotificationPromptSource.SquadPostModal,
```

Comment:
Not sure how this affects the Squads page. It should have a different `source` value.

Reply:
Oh wait did we introduce this so it only shows on squad post modal, not normal post modals?
If so, the solution is to split the two!

Basically now we had the exclusion where it could ALWAYS show on squad postr modal, even if you enabled already.

Reply:
I think this is the other way around, the logic is negated down below from this code. So it would always show except for that source.

Reply:
Maybe you can have a check with Tom in that case.
The issue is that notifications sometimes show even if you already accepted.

I think it has mainly to do with this logic, but you might be able to debug it better as you know it a bit better in detail.

Reply:
@sshanzel After investigating, do you suggest we leave this as it is currently or do this change?

Reply:
@sshanzel i think it was also a team member who checked and 100% already gave permissions.
To me this particular code did look like it would be able to fire even when you already have them enabled wouldnt you think?

Reply:
Hmm. After carefully evaluating each variable, it seems it should only display as necessary. Can you specify which ones that raise doubt so we can investigate it further?